oap-ape/ape-java/ape-common/src/main/java/com/intel/ape/util/Platform.java

```diff
@@ -1,337 +0,0 @@
-/*
```

Comment:
Why remove this file? I remember in other modules like ape-spark will use this class.

Reply:
I have put this file in the ICL library. for ape-spark, we can import in the one in Spark or the one in ICL library.

Reply:
emmm, this means you need icl related jar even if you didn't use compress related feature. And build will need such dependency which are not public on maven central repo.

Reply:
Yes. Currently we have add the ICL jar as dependency. Any way to avoid this?

Reply:
I suggest to keep this file in both ape-common module and icl-java lib.

Reply:
Let's keep it since we decided to integrate ICL codec in APE project for now.